docs/DeveloperGuide.md

```diff
@@ -434,7 +434,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | power user    | view a list of all events hosted by the CCA | I can see event history                                                           |
 | `* * *`  | beginner user | add a member to an event                    | I can track which members are participating in the event                          |
 | `* * *`  | beginner user | remove a member from an event               | I can amend adding the wrong member to an event                                   |
-| `* * *`  | beginner user | view the members who attended an event      | I can estimate the number of members who will attend similar events in the future |
+| `* * *`  | beginner user | view the members who enrolled in an event   | I can estimate the number of members who will attend similar events in the future |
```

Comment:
If we change this we should change the "so that I can" portion to `I can estimate the number of members who will enrol in similar events in the future`

Reply:
Agree with both, will amend accordingly